package.json

```diff
@@ -36,8 +36,8 @@
     "start.deving": "gulp start.deving --color",
     "tasks.list": "gulp --tasks-simple --color",
     "test": "gulp test --color",
-    "e2e.ci": "gulp build.prod --color && gulp build.js.e2e --color && gulp e2e --color",
-    "tests.all": "npm test && npm run e2e.ci",
+    "e2e.ci": "gulp build.prod --color && gulp build.e2e --color && gulp e2e --color",
```

Comment:
build.e2e runs clean.e2e as well. I think we had the cause that a test case has been renamed and build.js.e2e does not clean the old transpiled files